docs/pre-release/reference/intercepts/manual-agent.md

```diff
@@ -0,0 +1,217 @@
+import Alert from '@material-ui/lab/Alert';
+
+# Manually injecting the Traffic Agent
+
+It is possible to directly modify your workload's YAML configuration to add the Telepresence Traffic Agent and enable it to be intercepted.
+
+<Alert severity="warning">
+This is not the recommended approach for making a workload interceptable, but is sometimes the only possible approach.
```

Comment:
I _kinda_ get it.
I still want to make the introduction a little clearer on context and alternatives before dipping it. Let me know if this is approximately usable:

"When you use a Telepresence intercept, Telepresence automatically edits the workload and services when you use `telepresence uninstall --agent <your_agent_name>`.  In some GitOps workflows, you may need to use the [Telepresence Mutating Webhook] (../cluster-config/#mutating-webhook) to keep intercepted workloads unmodified while you target changes on specific pods.

In situations where you don't have access to the proper permissions for numeric ports, as noted in the [Note on numeric ports](../cluster-config/#note-on-numeric-ports) section of the documentation, it is possible to manually inject the Traffic Agent. Because this is not the recommended approach to making a workload interceptable, try the [Mutating Webhook](../cluster-config/#mutating-webhook) before proceeding."